frontend/src/store/modules/website/actions.js

```diff
@@ -24,10 +31,15 @@ export default {
     [FETCH_CURRENT_WEBSITE]: (context) => {
         context.commit(SET_FETCH_TRUE);
         const id = context.state.currentWebsite.id;
+
         return getCurrentUserWebsite(id).then(response => {
             context.commit(SET_CURRENT_WEBSITE, response.data);
-        }).catch(() => {
+            if(id === 0) {
```

Comment:
What is this? Why you ever fetch something with id === 0?

Reply:
We have to return current website id to backend for getting data. After user login there is no current website in the state. That's why we return 0. At the backend if id=0 we return  website where user is owner or first existing where user is member

Reply:
It's totally wrong, if you pass 0, endpoint should return website with id 0. Also, that is why I said, that everything that related to website fetching should be revisited

Reply:
in this context the check is for setting selected item in drop down menu as default

Reply:
Again, you fetching data by ID === 0!